### ISSUE TITLE
Using round robin algo for sending events to kafka

### ISSUE THREAD

**Issue by amitdaniel86** [pcfens/puppet-filebeat#99]
Hi , 

I would like to use the round_robin algo setting for output.kafka .

For an example i want to add: 

```
round_robin:
     group_events: 10
     reachable_only: true
```
I tried to do something like : 

```
class { 'filebeat':
      major_version  => '5',
      package_ensure => '5.2.2-1',
      outputs        => {
        'kafka'     => {
        'hosts'     => [
            'localhost:9092',
          ],

        'round_robin' => {
            'group_events' => '10',
            'reachable_only' => 'true',
            },
            'enabled' => true,
            'topic'   => 'blabla',
            'version' => '0.9.0',
        },
      },
    }
```
But the round_robin section is not getting parsed to the filebeat.yml config file.

Is it possible to do that on 0.10.4 version ?

**Comment by pcfens** [comment]
Is this fully resolved in your PR #100? It looks like reachable_only isn't still configurable.

**Comment by amitdaniel86** [comment]
@pcfens Should work with my PR .
why do you think it's not configured ?

**Comment by pcfens** [comment]
That piece didn't make it in the template.

**Comment by amitdaniel86** [comment]
Yep , my mistake .

**Comment by amitdaniel86** [comment]
in your template you are already defined the reacheable_only settings ... so i can use it no need to define it twice .

**Comment by amitdaniel86** [comment]
@pcfens when do you think you're going to build a new version with this change ?

**Comment by pcfens** [comment]
Hopefully soon (this afternoon or tomorrow). Because of the number of breaking changes in this release I want to make sure sure I document them really well.

I'm going to go ahead and close this since it's fixed in #100
